include/NovelRT/Transform.h

```diff
@@ -8,41 +8,74 @@
 #endif
 
 namespace NovelRT {
+  /**
+   * Represents a 2D position, scale and rotation.
+   */
   class Transform {
   protected:
     Maths::GeoVector2<float> _position;
     Maths::GeoVector2<float> _scale;
     float _rotation;
 
   public:
+    /**
+     * Creates a new Transform, with the specified position, rotation and scale.
```

Comment:
Grammar nit: Don't need the first comma here.

Reply:
Forgot to delete it